_includes/v22.2/metric-names.md

```diff
@@ -247,6 +247,7 @@ Name | Help
 `sql.txn.abort.count` | Number of SQL transaction ABORT statements
 `sql.txn.begin.count` | Number of SQL transaction BEGIN statements
 `sql.txn.commit.count` | Number of SQL transaction COMMIT statements
+`sql.txn.contended.count` | Number of SQL transactions experienced contention
```

Comment:
Nit: `Number of SQL transactions that experienced contention`

Reply:
Metrics on this page are (or rather: will shortly be) auto-generated based on `chartcatalog` output. As such, manual efforts are overwritten by verbatim upstream each time I update this file (at each minor rev, or per ticket adding new metric: whichever sooner).

I'll edit this one by hand, as we're not yet fully automated here. But the real (only) fix is to get engineering to revisit these descriptions. [Awaiting upstream response](https://github.com/cockroachdb/cockroach/issues/92925)!

Reply:
Oh, I see! Thank you!